participants/Zangshell virtual reality.md

```diff
@@ -0,0 +1,102 @@
+
+*Give us a brief description of what your application/UI does and how users are intended to interact with it.*
+
+The users are able to watch VR/AR image or videos on PC browsers or smartphone browsers.people can not only zoom in or zoom out a scene,but also can view virtual reality from different angles by using a PC mouse or gestures on a mobile screen.
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*You can reference the [list of curated datasets in this repo](https://github.com/filecoin-project/slingshot/blob/master/datasets.md). If you'd like to use a dataset that is not on that list, please nominate your chosen dataset by updating the curated datasets table in this PR. If nominating a dataset, do you own or have exclusive rights to the data you plan to use?*
+
+Not curated dataset.
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*What sorts of data will you collect? Do you own or have exclusive rights to the data you plan to use? Is the data license available online? Does the data contain any sensitive or illegal information?*
+
+We will collect orginal video data from a kind of dedicated cameras. and these video files are uploaded to IPFS after VR processing.We are authorized to have exclusive rights to the data we plan to use.the data license is availabe online. No license required, No sensitive and illegal information. 
+
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*Give us as realistic an estimate as you possibly can at this stage. We understand it's impossible to have perfect information before really starting your project!*
+
+about 50TB.
+
+## (7) How are you structuring the data?
+
+*Is this structured or unstructured data? What data formats, schemas, etc. does this dataset follow?*
+
+Those video data is Unstructured data, the common data formats are Universal Scene Description（USDZ）or MP4 or jpg etc.usually We will convert most video files in MP4 file format.
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+*How will you prepare this data for ingestion into Filecoin? What size storage deals will you be making? If this is a tabular dataset or directory structure, how will you maintain indices into the data so you can retrieve specific data as needed for your application? This is currently one of the most important steps to being successful with data storage on Filecoin. Feel free to ask for help and/or look at docs for recommendations!*
+
+just download the video files from cameras, after processing by VR tools,the data files could be coded in MP4 or USDZ format.usually,each video file size is less than 500MB.so each file could be sealed into the IPFS sectors smoothly.
+
+Not tabular dataset or directory structure.
+
+## (9) What tech stack will you use to store the data?
+
+*Either:* lotus
```

Comment:
For hot retrieval from the Filecoin network, you may want to consider IPFS or Powergate.